src/ansys_sphinx_theme/__init__.py

```diff
@@ -285,6 +286,92 @@ def update_footer_theme(
     context["ansys_sphinx_theme_version"] = __version__
 
 
+def add_cheat_sheet(
+    app: Sphinx, pagename: str, templatename: str, context: Dict[str, Any], doctree: document
+) -> None:
+    """Add a cheatsheet to the sidebar.
+
+    Parameters
+    ----------
+    app : ~sphinx.application.Sphinx
+        Application instance for rendering the documentation.
+    pagename : str
+        The name of the current page.
+    templatename : str
+        The name of the template being used.
+    context : dict
+        The context dictionary for the page.
+    doctree : ~docutils.nodes.document
+        The doctree
```

Comment:
```suggestion
        Name of the current page.
    templatename : str
        Name of the template being used.
    context : dict
        Context dictionary for the page.
    doctree : ~docutils.nodes.document
        The doctree.
```
The description for the last bullet should be in plain English, without the article "The" at the beginning. I wasn't sure what you meant to suggest anything.